refs.bib

```diff
@@ -0,0 +1,8 @@
+@article{Doe2016,
+  author = {Doe, J and Smith, R},
+  journal = {Title of the journal},
+  pages = {10--30},
+  title = {Title of the paper},
```

Comment:
Thanks for the comment. Interesting! 

In my own refs.bib file, most of the titles does indeed have double curly braces. However, as you noted in your edit it seems to be more complex, as I also found occurrences of titles with single curly braces. Both versions (double and single curly braces) render as I intended them to, so:

this works: `title = {Some random paper about a SPECIFIC sensor}`
but this also works: `title = {{Some random paper about LiDAR}}`